VideoPlayer/VideoPlayer.js

```diff
@@ -1551,7 +1561,9 @@ const VideoPlayerBase = class extends Component {
 	};
 
 	/**
-	 * Changes the playback speed.
+	 * Rewind to play the current media for seeking.
+	 * It is changes the playback rate.
+	 * If play during rewinding, playback speed will be update to normal.
```

Comment:
or how about "If you play or pause during rewinding, the playback speed updates to normal." ?

Reply:
I updated to "If you play or pause during rewinding, the playback speed updates to normal". I think it would be better to write a description so that it is not limited to button presses. The same goes for the comments below.